code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -402,6 +402,14 @@
 	to_chat(S, "<span class='warning'>This object does not contain enough materials to work with.</span>")
 	return FALSE
 
+/obj/structure/closet/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(!opened)
+		for(var/mob/living/L in contents)
+			if(!issilicon(L) && !isbrain(L))
+				to_chat(S, "<span class='warning'>An organism has been detected inside this object. Aborting.</span>")
+				return FALSE
+	..()
+
```

Comment:
This feels like you could probably standardise this for all /obj, just to check for /mob/living in the GetContents(), which would cover human operated machinery and all that.